src/sd.jl

```diff
@@ -1,4 +1,44 @@
-# Used in @constraint model x in PSDCone
+"""
+    PSDCone
+
+Positive semidefinite cone object that can be used to constrain a square matrix
+to be positive semidefinite in the [`@constraint`](@ref) macro. If the matrix
+has type `Symmetric` then the columns vectorization of its upper triangular
```

Comment:
The "columns vectorization" concept is used a lot in this docstring. Consider spending a sentence defining what it is.